### DISCUSSION TITLE
Welcome to nfsen Discussions!

### DISCUSSION THREAD

**Discussion by phaag** [phaag/nfsen#10] [Announcements]
## Welcome to discussions for nfsen!
  Use Discussions as a place to connect with other members of the nfsen community. This should replace the nfsen-general mailing list. Please:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.

**Comment by amxx70c** [comment]
Hello!

I use nfsen-1.3.8.

The problem: nfsen incorrectly deletes files with this pattern - nfcapd.current.* - when "%sources" in nfsen.conf have "'port' => '0'", for example:

%sources=(
    'test' => { 'IP' => '192.0.2.1',  'port' => '0', 'col' => '#000080', 'type' => 'netflow' },
);

For the above configuration nfsen must not start/stop the collector because the last is managed outside nfsen. And when nfsen deletes nfcapd.current.* the collector cannot save current netflow data because of deletion of the appropriate file by nfsen.

I propose to not delete the collector's current stat file for the above configuration. The working patch is attached.
[NfSen.pm.patch](https://github.com/phaag/nfsen/files/13557297/NfSen.pm.patch)
